### PR TITLE
LibWeb: Stub out all the functions from the execCommand spec

### PR DESCRIPTION
**Pull request by ADKaster**
Per the specification, it's ok if we say that nothing is supported.

It's not ok if we say something is supported but do nothing, apparently.

First step towards #23176

### CODE REVIEW

File: Userland/Libraries/LibWeb/DOM/Document.cpp [hunk 1]
```diff
@@ -3857,4 +3850,40 @@ Vector<JS::NonnullGCPtr<Element>> Document::elements_from_point(double x, double
     return sequence;
 }
 
+// https://w3c.github.io/editing/docs/execCommand/#execcommand()
+bool Document::exec_command(String, bool, String)
+{
+    return false;
+}
+
+// https://w3c.github.io/editing/docs/execCommand/#queryCommandEnabled()
```
Comment (trflynn89) [inline Userland/Libraries/LibWeb/DOM/Document.cpp:3859]: Looks good, but this URL and the ones that follow don't work - the anchors should all be lowercase, e.g.:
`https://w3c.github.io/editing/docs/execCommand/#querycommandenabled()`

Comment (ADKaster) [reply Userland/Libraries/LibWeb/DOM/Document.cpp:3859]: 😬 that's what I get for letting CoPilot generate the URLs instead of copying them from the spec

**Comment by trflynn89** [comment]
You've got ~~mail~~ conflicts :(

**Review by trflynn89 — APPROVED** [review]

**Comment by trflynn89** [comment]
Conflict again after merging your other PR

**Pull request merged**